rtt_roscomm/src/passthrough_callback_queue.cpp

```diff
@@ -0,0 +1,24 @@
+
```

Comment:
nit: remove empty line and add a license header instead

Reply:
Same as before, I prefer not to add the license header.